### PR TITLE
339 3.29 email address profile

### PR DESCRIPTION
**Pull request by Baschdl**
Add the email address to the profile. We don't need extra abilities because only the organizer and admin can view the profiles of other persons.
#339

### CODE REVIEW

File: spec/features/profile_spec.rb [hunk 1]
```diff
@@ -137,4 +137,4 @@ def mock_writing_to_filesystem
 
     expect(page).to have_css(".has-error", count: 3)
   end
-end
+end
```
Comment (bjrne) [inline spec/features/profile_spec.rb]: Why did you remove the trailing newline here?

**Review by bjrne — APPROVED** [review]
Good, but please add trailing newlines in both files 🏒

**Review by ekrebs5 — APPROVED** [review]

File: spec/features/profile_spec.rb (hunk 1 above)
Comment (Baschdl) [reply spec/features/profile_spec.rb]: Fixed it
